plugins/system_upgrade.py

```diff
@@ -62,16 +62,20 @@
 CANT_RESET_RELEASEVER = _(
     "Sorry, you need to use 'download --releasever' instead of '--network'")
 
-STATE_VERSION = 2
+STATE_VERSION = 3
 
 # --- Miscellaneous helper functions ------------------------------------------
 
 
-def reboot():
+def reboot(poweroff_after = None):
```

Comment:
I would use just `poweroff` here. The `_after` suffix is not relevant in this context.

Reply:
Also we could use just the default = False.